portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import java.lang.*; 
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns comments data */
+
+@WebServlet("/comments")
+public class CommentsServlet extends HttpServlet {
+    private final List<String[]> comments = new ArrayList<>();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("application/json");
+        String json = new Gson().toJson(comments);
+        response.getWriter().println(json);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+            String[] comment = new String[2];
+            comment[0] = "name: " + request.getParameter("fullname");
```

Comment:
Consider making the comment a structured piece of data on the server.

Here, you're having the server decide how the data will be displayed to the user. Instead, the server should provide the data in a structured format, and the client (i.e. the JS code) should decide how to display it.

You can do this by defining a Java class with fields for 'name' and 'comment'.

Reply:
Hi Erick, I did that in the next PR that's going to come :)

Reply:
sounds good.